combat-trainer.lic

```diff
@@ -1171,6 +1181,15 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each do |name|
+      if !(Flags["ap-#{name}-active"] == true)
```

Comment:
Not really grasping how to do this then. I set the string in the add. I then set the flag to true if the bput is successful with activation. I then check each iteration to see if it is not true and then trigger the spell off that. Wouldn't I need to then trigger it like `if Flag text.equals? current_buff_being_checked[expired_message]`?